feature-terminate-insurance/src/main/kotlin/com/hedvig/android/feature/terminateinsurance/data/TerminationStep.kt

```diff
@@ -0,0 +1,19 @@
+package com.hedvig.android.feature.terminateinsurance.data
+
+import kotlinx.datetime.LocalDate
+
+sealed interface TerminationStep {
+  class Date(
+    val minDate: LocalDate,
+    val maxDate: LocalDate,
+  ) : TerminationStep
+
+  class Success(
+    val terminationDate: LocalDate,
+    val surveyUrl: String,
+  ) : TerminationStep
+
+  class Failed(
+    val message: String?,
+  ) : TerminationStep
```

Comment:
I think it's a good idea to make those `data class` to get the free `equals` implementation out of them. Especially if these ever reach down to compose levels and they need to be compared it's nice to get it for free.